dev-libs/jansson/jansson-2.13.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib-minimal
+
+DESCRIPTION="C library for encoding, decoding and manipulating JSON data"
+HOMEPAGE="https://www.digip.org/jansson/"
+SRC_URI="https://www.digip.org/jansson/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="doc static-libs"
+
+BDEPEND="doc? ( >=dev-python/sphinx-1.0.4 )"
+RDEPEND=""
+
+multilib_src_configure() {
+	ECONF_SOURCE="${S}" econf $(use_enable static-libs static)
+}
+
+multilib_src_compile() {
+	default
+
+	if multilib_is_native_abi && use doc ; then
+		emake html
```

Comment:
```
make -j16 -l16 html 
Making html in doc
make[1]: Entering directory '/var/tmp/portage/dev-libs/jansson-2.13.1/work/jansson-2.13.1-abi_x86_64.amd64/doc'
sphinx-build -b html -d _build/doctrees  /var/tmp/portage/dev-libs/jansson-2.13.1/work/jansson-2.13.1/doc _build/html
Running Sphinx v3.1.1

Exception occurred:
  File "/var/tmp/portage/dev-libs/jansson-2.13.1/work/jansson-2.13.1/doc/ext/refcounting.py", line 60, in setup
    app.add_directive('refcounting', refcounting_directive, 0, (1, 0, 0))
TypeError: add_directive() takes from 3 to 4 positional arguments but 5 were given
The full traceback has been saved in /var/tmp/portage/dev-libs/jansson-2.13.1/temp/sphinx-err-1gdlk6p_.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make[1]: *** [Makefile:453: html-local] Error 2
make[1]: Leaving directory '/var/tmp/portage/dev-libs/jansson-2.13.1/work/jansson-2.13.1-abi_x86_64.amd64/doc'
make: *** [Makefile:638: html-recursive] Error 1
 * ERROR: dev-libs/jansson-2.13.1::testworld failed (compile phase):
 *   emake failed
```

```
>: cat /var/tmp/portage/dev-libs/jansson-2.13.1/temp/sphinx-err-1gdlk6p_.log
# Sphinx version: 3.1.1
# Python version: 3.7.7 (CPython)
# Docutils version: 0.16 release
# Jinja2 version: 2.11.2
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/sphinx/cmd/build.py", line 279, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/usr/lib/python3.7/site-packages/sphinx/application.py", line 244, in __init__
    self.setup_extension(extension)
  File "/usr/lib/python3.7/site-packages/sphinx/application.py", line 398, in setup_extension
    self.registry.load_extension(self, extname)
  File "/usr/lib/python3.7/site-packages/sphinx/registry.py", line 419, in load_extension
    metadata = setup(app)
  File "/var/tmp/portage/dev-libs/jansson-2.13.1/work/jansson-2.13.1/doc/ext/refcounting.py", line 60, in setup
    app.add_directive('refcounting', refcounting_directive, 0, (1, 0, 0))
TypeError: add_directive() takes from 3 to 4 positional arguments but 5 were given
```

Reply:
Apparently sphinx was version bumped in between when i opened this PR and now, and apparently the new version breaks API compatibility. Changing the version requirements to something before sphinx 3 until the issue is fixed upstream.